### PR TITLE
RCHAIN-2307 Remove Task from RuntimeManger usage spots

### PR DESCRIPTION
**Pull request by dzajkowski**
## Overview
<sup>_What this PR does, and why it's needed_</sup>
Final step to F instead of Task for RuntimeManager.

### JIRA ticket:
<sup>_Create it if there isn't one already._</sup>
RCHAIN-2307

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [ ] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Comment by dzajkowski** [comment]
bros try

**Comment by dzajkowski** [comment]
bors try

**Comment by dzajkowski** [comment]
bors try

**Comment by dzajkowski** [comment]
please follow the commits. Ive been told that it makes it easier to follow.

**Comment by dzajkowski** [comment]
`continuous-integration/drone/pr — the build failed` is an sbt failure

File: casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala [hunk 1]
```diff
@@ -314,4 +318,61 @@ object RuntimeManager {
...
+          channel: Par
+      ): T[F, scala.Seq[Par]] = runtimeManager.getData(hash, channel).liftM[T]
+
+      override def getContinuation(
+          hash: ByteString,
+          channels: scala.collection.immutable.Seq[Par]
+      ): T[F, scala.Seq[(scala.Seq[BindPattern], Par)]] =
+        runtimeManager.getContinuation(hash, channels).liftM[T]
+
+      override def emptyStateHash: ByteString = runtimeManager.emptyStateHash
```
Comment (KentShikama) [inline casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala]: So you don't need to lift this one because it is a field?

Comment (dzajkowski) [reply casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala]: yup.

Comment (dzajkowski) [reply casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala]: maybe a better answer would be: it's a constant.

Comment (dzajkowski) [reply casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala]: I can replace the def with a val.

**Comment by dzajkowski** [comment]
bors try

**Pull request merged**